src/lib389/lib389/cli_idm/client_config.py

```diff
@@ -269,6 +282,23 @@ def display(inst, basedn, log, args):
         group_member='member',
     ))
 
+    log.info(DISPLAY_TEMPLATE.format(
```

Comment:
Are you sure about log.info here? IIRC info as a level is always printed from the CLI so maybe this should be debug?